BackgroundGeolocation/MAURPostLocationTask.m

```diff
@@ -152,6 +152,16 @@ - (BOOL) post:(MAURLocation*)location toUrl:(NSString*)url withTemplate:(id)loca
             }
         });
     }
+
+    if (statusCode == 401)
+    {   
+        dispatch_async(dispatch_get_main_queue(), ^{
+            if (_delegate && [_delegate respondsToSelector:@selector(postLocationTaskHttpAuthorizationUpdates:)])
```

Comment:
I would rather call postLocationTaskHttpAuthorizationUpdates method then duplicate it's functionality, but I see you took inspiration from another similar method call above.